src/index.ts

```diff
@@ -364,19 +380,46 @@ export default function fetchHAR(har: Har, opts: FetchHAROptions = {}) {
         options.body = request.postData.text;
       }
     }
+
+    /**
+     * The fetch spec, which Node 18+ strictly abides by, now requires that `duplex` be sent with
+     * requests that have payloads.
+     *
+     * As `RequestInit#duplex` isn't supported by any browsers, or even mentioned on MDN, we aren't
+     * sending it in browser environments. This work is purely to support Node 18+ and `undici`
+     * environments.
+     *
+     * @see {@link https://github.com/nodejs/node/issues/46221}
+     * @see {@link https://github.com/whatwg/fetch/pull/1457}
+     * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/Request/Request}
+     */
+    if (shouldSetDuplex && !isBrowser()) {
```

Comment:
is there any downside to passing this opt in browser environments? what if browsers start supporting it?

Reply:
Though the browser tests were passing with this data coming to them I'd rather be cautious with this and not send it. And as it's a breaking change in `undici` for them to start erroring if it's not present I can't see browsers also doing the same so we should be okay here to not send it yet.